ios/RNCCameraRollManager.m

```diff
@@ -70,7 +70,7 @@ + (PHFetchOptions *)PHFetchOptionsFromMediaType:(NSString *)mediaType
   }
   if (toTime > 0) {
     NSDate* toDate = [NSDate dateWithTimeIntervalSince1970:toTime/1000];
-    [format addObject:@"creationDate < %@"];
+    [format addObject:@"creationDate <= %@"];
```

Comment:
Just a note: while this changes behavior some, this unifies the logic on Android (https://github.com/react-native-community/react-native-cameraroll/blob/master/android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java#L331) and iOS, so is probably a net positive in the long run

Reply:
Added a note in the initial commit message!